foreshadow/intents/numeric.py

```diff
@@ -45,3 +47,31 @@ def transform(self, X, y=None):
 
         """
         return X.apply(pd.to_numeric, errors="coerce")
+
+    @classmethod
+    def column_summary(cls, df):  # noqa
+        data = df.iloc[:, 0]
+        nan_num = int(data.isnull().sum())
+        invalid_num = int(
+            pd.to_numeric(df.iloc[:, 0], errors="coerce").isnull().sum()
```

Comment:
Could you clarify what you mean by "show some samples"? Looking the examples provided by the `to_numeric` method, the most common case is `s = pd.Series(['apple', '1.0', '2', -3])` and `'apple'` cannot be converted to a number. We will need to turn back on quite a few unit tests to cover these changes since they are all marked as skip right now.